src/install.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Install
+ * A simple FIFO queue for installing plugins
+ */
+
+export default function install ([plugin, ...tail], app, callback) {
+  if (!plugin) return callback()
+
+  plugin.register(app, app._options, function(err) {
+    if (err) throw err
+    install(tail, app, callback)
```

Comment:
I didn't get this at first but now I see — you're recursively calling install for each plugin in the plugin array. Is there a need for the recursion, or is it just fun?